scripts/pre-commit.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+echo "Running detekt check..."
+OUTPUT="/tmp/detekt-$(date +%s)"
+./gradlew detektAll > $OUTPUT
```

Comment:
So I guess this will only work when run from the root folder right? I think it's probably ok, though.

Reply:
Git will change the current working dir when this script executes so I think it's fine:

> Before Git invokes a hook, it changes its working directory to either $GIT_DIR in a bare repository or the root of the working tree in a non-bare repository. An exception are hooks triggered during a push (pre-receive, update, post-receive, post-update, push-to-checkout) which are always executed in $GIT_DIR.

https://git-scm.com/docs/githooks